jujugui/static/gui/src/app/components/entity-content/entity-content-revisions.js

```diff
@@ -0,0 +1,98 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2015 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+YUI.add('entity-content-revisions', function() {
+
+  juju.components.EntityContentRevisions = React.createClass({
+
+    /**
+      Handle clicks on accordion. Toggles the concealed class on the list node
+      which hides all the list items apart from the first.
+
+      @method _handleAccordionClick
+    */
+    _handleAccordionClick: function(show) {
+      var list = ReactDOM.findDOMNode(this).querySelector('.revisions__list');
+      list.classList.toggle('list--concealed');
+    },
+
+    /**
+      Format ISO date/time to human readable format (2015-09-05).
+
+      @method _formatDate
+    */
+    _formatDate: function(iso) {
+      var date = new Date(Date.parse(iso));
+      date = date.getFullYear() + '-' +
+        ('0' + (date.getMonth() + 1)).slice(-2) + '-' +
+        ('0' + (date.getDay() + 1)).slice(-2);
+      return date;
+    },
+
+    /**
+      Generate the list of revisions.
+
+      @method _generateRevisions
+      @return {Object} The revisions markup.
+    */
+    _generateRevisions: function() {
+      var components = [];
+      var revisions = this.props.revisions;
+      revisions.forEach(function(revision) {
```

Comment:
you can use an arrow function here instead of relying on the second param to bind the context

```JavaScript
revisions.forEach((revision) => { 
```